app/command/v2/put_content.rb

```diff
@@ -30,6 +30,22 @@ def metadata
     content_item.except(*content_item_top_level_fields)
   end
 
+  def link_set
+    @link_set ||= LinkSet.find_by(content_id: content_id)
+  end
+
+  def link_set_hash
+    if link_set.present?
+      {links: link_set.links.deep_symbolize_keys}
```

Comment:
We might not need to `#deep_symbolize_keys` here because `include SymbolizeJSON` appears in the LinkSet model.

This will symbolize keys automatically when they're ready from the model.